projects/demo/src/modules/components/editor/examples/10/index.ts

```diff
@@ -0,0 +1,40 @@
+import {ChangeDetectorRef, Component, Inject} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {tuiRawLoad} from '@taiga-ui/addon-doc';
+import {TUI_EDITOR_CONTENT_PROCESSOR, TuiEditorTool} from '@taiga-ui/addon-editor';
+import MarkdownIt from 'markdown-it';
+
+@Component({
+    selector: `tui-editor-example-10`,
+    templateUrl: `./index.html`,
+    styleUrls: [`./index.less`],
+    providers: [
+        {
+            provide: TUI_EDITOR_CONTENT_PROCESSOR,
+            useValue: (markdown: string): string => new MarkdownIt().render(markdown),
+        },
+    ],
+    changeDetection,
+    encapsulation,
+})
+export class TuiEditorExample10 {
+    readonly builtInTools = [
+        TuiEditorTool.Undo,
+        TuiEditorTool.Img,
+        TuiEditorTool.Link,
+        TuiEditorTool.Anchor,
+    ];
+
+    control?: FormControl;
+
+    constructor(@Inject(ChangeDetectorRef) private readonly cd: ChangeDetectorRef) {}
+
+    async ngOnInit(): Promise<void> {
+        const md = await tuiRawLoad(import(`./example.md?raw`));
```

Comment:
I think it makes more sense to create control initially and patch value when it's loaded